lib/components/TokenDetailsCard.jsx

```diff
@@ -105,7 +105,7 @@ export const TokenDetailsCard = (props) => {
       <Collapse title='Advanced Settings' className='mt-4 sm:mt-8'>
         <InputLabel
           secondary='Pool Sponsorship Ticket'
-          description='Provide a name and ticker symbol for the ERC20 token that will be created and used as the pools sposnorship tickets. Sponsorship tickets are not eligible to win prizes.'
+          description={`Provide a name and ticker symbol for the ERC20 token that will be created and used as the pool's sposnorship tickets. Sponsorship tickets are not eligible to win prizes.`}
```

Comment:
Just noticed this also says Sposnorship (instead of Sponsorship) the first time.